platformio/src/renderer.cpp

```diff
@@ -917,46 +918,53 @@ void drawStatusBar(String statusStr, String refreshTimeStr, int rssi,
                    double batVoltage)
 {
   String dataStr;
+  uint16_t dataColor = GxEPD_BLACK;
   display.setFont(&FONT_6pt8b);
   int pos = DISP_WIDTH - 2;
   const int sp = 2;
 
   // battery
   int batPercent = calcBatPercent(batVoltage);
+  if (batPercent < calcBatPercent(BATTERY_WARN_VOLTAGE)) {
+    dataColor = ACCENT_COLOR;
+  }
   dataStr = String(batPercent) + "% (" 
             + String( round(100.0 * batVoltage) / 100.0, 2 ) + "v)";
-  drawString(pos, DISP_HEIGHT - 1 - 2, dataStr, RIGHT);
+  drawString(pos, DISP_HEIGHT - 1 - 2, dataStr, RIGHT, dataColor);
   pos -= getStringWidth(dataStr) + 25;
   display.drawInvertedBitmap(pos, DISP_HEIGHT - 1 - 17,
-                             getBatBitmap24(batPercent), 24, 24, GxEPD_BLACK);
+                             getBatBitmap24(batPercent), 24, 24, dataColor);
   pos -= sp + 9;
 
   // wifi
   dataStr = String(getWiFidesc(rssi));
+  dataColor = getWiFiColor(rssi, GxEPD_BLACK, ACCENT_COLOR);
```

Comment:
Can we simplify this to something like this...
`dataColor = rssi >= -70 ? GxEPD_BLACK : ACCENT_COLOR'`